Library/Homebrew/cmd/audit.rb

```diff
@@ -855,6 +855,14 @@ def audit_line(line, lineno)
       if line =~ /(require ["']formula["'])/
         problem "`#{$1}` is now unnecessary"
       end
+
+      if line =~ /#\{share\}\/#{formula.name}/
+        problem "Use \#{pkgshare}\ instead of \#{share}\/#{formula.name}"
+      end
+
+      if line =~ /share\/"#{formula.name}"/
```

Comment:
Same like in the other case, i.e. use `String#include?` for speed and simplicity.